porch/repository/pkg/git/git.go

```diff
@@ -496,6 +515,58 @@ func parseDraftName(draft *plumbing.Reference) (name, revision string, err error
 	return name, revision, nil
 }
 
+func (r *gitRepository) loadTaggedPackages(tag *plumbing.Reference) ([]repository.PackageRevision, error) {
+	name := tag.Name().String()
+	if !strings.HasPrefix(name, refTagsPrefix) {
+		return nil, fmt.Errorf("invalid tag ref name: %q", name)
```

Comment:
I'm surprised that `go vet` isn't complaining about this.  Typically it gets annoyed when you have an `else` if you're returning from the `if-true` block

Reply:
odd indeed. I fixed it and using TrimPrefix as per above also